celer/lasso_fast.pyx

```diff
@@ -100,7 +100,10 @@ def celer(
     cdef int[:] all_features = np.arange(n_features, dtype=np.int32)
 
     for t in range(max_iter):
-        if t != 0:
+        # if t != 0:
```

Comment:
This introduces a severe regression a the beginning of paths for large X (e.g. finance) where X.T @ theta is recomputed while there are 3 features in the WS.

Maintaining Xtheta from one alpha to the other may be a fix.